source/includes/_orders.md.erb

```diff
@@ -485,6 +485,73 @@ curl "<%= config[:host] %>/orders/R314234460/promotions/12345" \
 }
 ```
 
+## Получение списка позиций заказа
+
+> Пример команды:
+
+```shell
+curl "<%= config[:host] %>/orders/R314234460/line_items" \
+  -H "Authorization: Token token=#{TOKEN}"
+```
+
+> Ответ будет содержать следующие данные:
+
+```json
+{
+  "line_items": [
+    {
+      "id": 1,
+      "quantity": 1,
+      "price": 1236.7,
+      "total": 1236.7,
+      "customer_comment": null,
+      "product_in_stock": true,
+      "product": {
+        "id": 1,
+        "sku": "20002",
+        "name": "Product #0",
+        "price": 1136.7,
+        "original_price": 1236.7,
+        "discount": 100,
+        "human_volume": "100 г. x 12",
+        "items_per_pack": 12,
+        "images": [
+          "mini_url": "http://instamart.ru/spree/products/5946/mini/10006.jpg?1473242617",
+          "small_url": "http://instamart.ru/spree/products/5946/small/10006.jpg?1473242617",
+          "product_url": "http://instamart.ru/spree/products/5946/product/10006.jpg?1473242617",
+          "preview_url": "http://instamart.ru/spree/products/5946/preview/10006.jpg?1473242617",
+          "original_url": "http://instamart.ru/spree/products/5946/original/10006.jpg?1473242617"
+        ]
+      }
+    }
+  ],
+  "meta": {
+    "current_page": 1,
+    "next_page": null,
+    "total_pages": 1,
+    "per_page": 10
+  }
+}
+```
+
+Получить все позиции заказа пользователя можно, выполнив запросы:
+
+`GET <%= config[:host] %>/orders/#{ORDER_NUMBER}/line_items`
+
+`GET <%= config[:host] %>/shipments/#{SHIPMENT_NUMBER}/line_items`
+
+### Структура объекта LineItem
+
+Атрибут | Описание
+--------- | -----------
+id | идентификатор
+quantity | количество товаров
+price | стоимость товара
+total | общая стоимость
+customer_comment | комментарий пользователя
+product_in_stock | товар есть в наличии
+product | данные о товаре
```

Comment:
Ensure that we don't duplicate existing description

Reply:
fixed
I don't know why I could not find a duplicate at first, although I tried